styles/style.css

```diff
@@ -0,0 +1,435 @@
+    font-weight: 400;
+}
+
+.left-menu .left-menu-link-text {
+    padding: 16px 0px;
+}
+
+.left-menu nav a:focus {
+    background-color: var(--third-color);
+    outline: none;
+}
+
+main {
+    height: 100vh;
+
+    border: 1px solid var(--third-color);
+
+    background-color: var(--second-color);
+
+    overflow-y: scroll;
+    /* Hide scrollbar */
+    -ms-overflow-style: none;  /* IE and Edge */
+    scrollbar-width: none;  /* Firefox */
+}
+
+main::-webkit-scrollbar {
+    /* Hides scrollbar for Chrome, Safari and Opera */
+    display: none;
+}
+
+main > * {
+    background-color: #ffffff;
+}
+
+main header {
+    position: sticky;
+    top: 0;
+
+    background-color: #ffffffe7;
+    backdrop-filter: blur(4px);
+
+    z-index: 1;
+}
+
+main header h1 {
+    margin: 0;
+
+    font-size: 20px;
+}
+
+main section img {
+    border-radius: 50%;
+}
+
+main .tweet-options button {
+    border-radius: 50%;
+    padding: 8px 8px 3px 8px;
+    margin-left: -10px;
+    margin-right: 6px;
+}
+
+main .tweet-options svg {
+    color: var(--main-color);
+}
+
+main .post {
+    width: 100%;
+
+    border-radius: 15px;
+}
+
+#tweet {
+    margin-bottom: 8px;
+}
+
+main #tweet p {
+    margin-top: 10px;
+
+    font-size: 20px;
+    color: var(--fourth-color);
+}
+
+main .feed {
+    font-size: 16px;
+}
+
+main .feed li {
+    padding: 8px 16px;
+}
+
+main .feed p {
+    line-height: 1.3;
+}
+
+main .feed svg {
+    color: var(--fourth-color);
+}
+
+main .feed span {
+    color: var(--fourth-color);
+}
+
+main .feed .tag {
+    color: var(--main-color);
+}
+
+main .feed button {
+    border-radius: 50%;
+
+    padding: 8px 8px 3px 8px;
+}
+
+main .tweet-options button:hover,
+main .feed button:hover {
+    background-color: #1ea1f31a;
+}
+
+main .feed button svg:hover {
+    fill: var(--main-color);
+}
+
+main .feed .retweet:hover {
+    background-color: #45db1f1a;
+}
+
+main .feed .retweet svg:hover {
+    fill: #47bb2a;
+}
+
+main .feed .like:hover {
+    background-color: #f918801a;
+}
+
+main .feed .like svg:hover {
+    fill: #f91880;
+}
+
+main .bottom-nav {
+    position: sticky;
+    bottom: 0;
+
+    margin-top: 12px;
+    padding-top: 8px;
+
+    z-index: 1;
+}
+
+main .bottom-nav a {
+    padding: 4px 2px 0px 4px;
+}
+
+.search-bar p {
+    font-size: 14px;
+    color: var(--fourth-color);
+}
+
+.side-bar-container {
+    background-color: var(--second-color);
+    border-radius: 20px;
+
+    width: 200px;
+}
+
+.side-bar-container h2 {
+    margin: 0;
+}
+
+.side-bar-container article > *,
+.side-bar-container ul li p {
+    font-size: 14px;
+}
+
+.side-bar-container img {
+    border-radius: 15px;
+}
+
+
+/* COMPONENT STYLES */
+
+#logo {
+    padding-left: 12px;
+}
+
+#tweet-button {
+    color: #ffffff;
+    font-size: 14px;
+    font-weight: 600;
+
+    padding: 12px 36px;
+    
+    background-color: var(--main-color);
+
+    border-radius: 35px;
+}
+
+#tweet-button:hover {
+    color: #ffffff;
+    background-color: #1ea1f398;
```

Comment:
How did you get this background-color?

Reply:
The `--main-color: #1ea1f3;` was taken from twitter using a chrome extension called "Eye Dropper". This specific color used here is just that main color with some transparency added to it.